whylog/tests/tests_client/__init__.py

```diff
@@ -25,24 +24,25 @@ class TestBasic(TestCase):
         '013_match_and_incomplete',
     )
     def test_one(self, test_name):
-        prefix_path = path_join(*path_test_files)
-        path = path_join(prefix_path, test_name)
-        parsers_path = path_join(path, 'parsers.yaml')
-        rules_path = path_join(path, 'rules.yaml')
-        input_path = path_join(path, 'input.txt')
-        output_path = path_join(path, 'expected_output.txt')
-        log_location_path = path_join(path, 'log_locations.yaml')
+        prefix_path = os.path.join(*test_paths.path_test_files)
+        path = os.path.join(prefix_path, test_name)
+        parsers_path = os.path.join(path, 'parsers.yaml')
+        rules_path = os.path.join(path, 'rules.yaml')
+        input_path = os.path.join(path, 'input.txt')
+        output_path = os.path.join(path, 'expected_output.txt')
+        log_location_path = os.path.join(path, 'log_locations.yaml')
 
         whylog_base = YamlConfig(
             parsers_path=parsers_path,
             rules_path=rules_path,
             log_locations_path=log_location_path,
         )
-        whylog_client = WhylogClient(base=whylog_base, open_path=path)
+        # whylog_client = WhylogClient(config=whylog_base, open_path=path)
 
         with open(input_path, 'r') as f:
             vim_line = f.read()
-        result = whylog_client.get_cause(vim_line)
+        # TODO call get_cause with sens...
+        # result = whylog_client.get_cause(1, vim_line)
```

Comment:
You can finish the test but annotate it with `SkipTest`. This way your indention to interface will become clear for the reader of the test, which really helps align the understanding of author and reviewer.